libpoolprotocols/stratum/EthStratumClient.h

```diff
@@ -56,9 +56,9 @@ class EthStratumClient : public PoolClient
 	void readResponse(const boost::system::error_code& ec, std::size_t bytes_transferred);
 	void processReponse(Json::Value& responseObject);
 
-	cred_t * p_active;
-	cred_t m_primary;
-	cred_t m_failover;
+	PoolConnection * p_active;
```

Comment:
i think this can be limited to only one

Reply:
Yep. Doesn't even need the pointer anymore.